src/libs/shim-services/src/shim_mgmt/client.rs

```diff
@@ -10,13 +11,13 @@
 
 use std::{path::Path, path::PathBuf, time::Duration};
 
-use super::server::mgmt_socket_addr;
+use crate::mgmt_socket_addr;
 use anyhow::{anyhow, Context, Result};
 use hyper::{Body, Client, Method, Request, Response};
 use hyperlocal::{UnixClientExt, UnixConnector, Uri};
 
 /// Shim management client with timeout
-pub struct MgmtClient {
+struct MgmtClient {
```

Comment:
Why `MgmtClient` is not public? What if other crate want to use this `struct`

Reply:
Fixed